immix/llvm/memory_manager.cpp

```diff
@@ -37,9 +35,13 @@ static uint8_t *roundTripAllocateDataSection(void *object, uintptr_t size,
 {
   auto alloc_addr = static_cast<SectionMemoryManager *>(object)->allocateDataSection(
       size, alignment, sectionID, sectionName, isReadOnly);
-  // printf("sec name: %s\n", sectionName);
-  auto mapName = "_GC_MAP_";
-  if (strncmp(sectionName, mapName, strlen(mapName)) == 0)
+// printf("sec name: %s\n", sectionName);
+#ifdef __APPLE__
```

Comment:
Does the `__APPLE__` macro represent the MacOs? Can document why we need different `llvmSectionName` to fit the OS.